source/agora/consensus/Ledger.d

```diff
@@ -718,6 +718,51 @@ public class ValidatingLedger : NodeLedger
         return null;
     }
 
+    /***************************************************************************
+
+        Override so that if this validator has not yet signed it is not included
+
+        Params:
+            header = header to check the signatures of
+            validators = validator info for the active validators
```

Comment:
Since we're using `Ditto` for the next overload, `height` also needs to be documented

Reply:
Updated